cfme/tests/ansible/test_embedded_ansible_crud.py

```diff
@@ -133,3 +106,38 @@ def test_embedded_ansible_logs(embedded_appliance):
     # Retrieving setup log file from list and asserting with length
     # Setup log file contains date/time string in it.
     assert "setup" in diff[0]
+
+
+@pytest.mark.tier(3)
+def test_embedded_ansible_disable(embedded_appliance):
+    """Tests whether the embedded ansible role and all workers have stopped correctly
+
+    Polarion:
+        assignee: sbulage
+        casecomponent: Ansible
+        caseimportance: critical
+        initialEstimate: 1/6h
+        tags: ansible_embed
+    """
+    assert wait_for(lambda: embedded_appliance.rabbitmq_server.running, num_sec=30)
+    assert wait_for(lambda: embedded_appliance.nginx.running, num_sec=30)
+    embedded_appliance.disable_embedded_ansible_role()
+
+    if not embedded_appliance.is_pod:
+        assert wait_for(
+            lambda: embedded_appliance.supervisord.status == "inactive", num_sec=180
+        )
+        assert wait_for(
+            lambda: embedded_appliance.rabbitmq_server.status == "inactive", num_sec=80
+        )
+        assert wait_for(
+            lambda: embedded_appliance.nginx.status == "inactive", num_sec=30
+        )
+    else:
+
+        @wait_for_decorator(num_sec=300)
+        def is_ansible_pod_stopped():
+            # todo: implement appropriate methods in appliance
```

Comment:
Required: Please fallow TODO guideline https://github.com/ManageIQ/integration_tests/blob/master/docs/guides/dev_guide.rst

Reply:
@digitronik thanks for pointing this out, I am gonna placing this method in `IPAppliance`.